sound/soc/sdw_utils/soc_sdw_utils.c

```diff
@@ -1203,6 +1308,22 @@ int asoc_sdw_parse_sdw_endpoints(struct snd_soc_card *card,
 				    !(dai_info->quirk_exclude ^ !!(dai_info->quirk & ctx->mc_quirk)))
 					continue;
 
+				/*
+				 * Don't check if sdca endpoint present if machine quirk is set
```

Comment:
Don't check the sdca endpoint presence if machine quirk is set.

Reply:
Or
Skip the sdca endpoint presence check if machine quirk is set.

Reply:
done